packages/pipeline-editor/src/PipelineService.tsx

```diff
@@ -95,8 +95,8 @@ export class PipelineService {
     pipeline: any,
     runtimeName: string
   ): Promise<any> {
-    console.log('Pipeline definition:');
-    console.log(pipeline);
+    // console.log('Pipeline definition:');
+    // console.log(pipeline);
```

Comment:
rather than comment this out we should either delete it, leave it uncommented, or make it debug instead